### PR TITLE
3317 - Moving to YARN and including the reactSVG code

### PR DESCRIPTION
**Pull request by aronchick**
most of the code is just s/npm/yarn/g and yarn files for building/etc (in config/ and scripts/).

there is also - webui/src/images/svg-image.tsx - for rendering SVGs properly.

### CODE REVIEW

File: ops/terraform/remote_files/scripts/install-node.sh [hunk 1]
```diff
@@ -122,7 +122,7 @@ function install-bacalhau-from-source() {
   sudo apt-get -y install --no-install-recommends jq nodejs make
   git clone --branch ${BACALHAU_BRANCH} https://github.com/bacalhau-project/bacalhau.git
   pushd bacalhau
-  pushd webui && npm install && popd
+  pushd webui && yarn install && popd
```
Comment (simonwo) [inline ops/terraform/remote_files/scripts/install-node.sh:125]: Has this been tested? I don't think yarn will be installed here.

File: .circleci/config.yml [hunk 2]
```diff
@@ -132,6 +132,223 @@ commands:
       - run: echo 'export PATH="/home/circleci/.local/bin:$PATH"' >> "$BASH_ENV"
 
 jobs:
+  build_cli:
```
Comment (frrist) [inline .circleci/config.yml:135]: Is this purely a mechanical change? Just moving this section of code up?

File: ops/aws/canary/README.md [hunk 3]
```diff
@@ -67,10 +67,10 @@ These are the resources that had to be created/updated manually outside of CDK:
...
 ## Useful CDK commands
 Keep in mind that you might need to pass your AWS profile and the stack name in some of these commands:
-* `npm run build`   compile typescript to js
-* `npm run postinstall` deletes cdk golang templates that can result in breaking go commands due to invalid file naming pattern
-* `npm run watch`   watch for changes and compile
-* `npm run test`    perform the jest unit tests
+* `yarn run build`   compile typescript to js
+* `yarn run postinstall` deletes cdk golang templates that can result in breaking go commands due to invalid file naming pattern
+* `yarn run watch`   watch for changes and compile
+* `yarn run test`    perform the jest unit tests
```
Comment (simonwo) [inline ops/aws/canary/README.md:73]: I don't think this CDK stuff has been changed to use yarn??

File: ops/repo_init.sh [hunk 4]
```diff
@@ -9,5 +9,5 @@ poetry install
 popd
 
 pushd webui
-npm install
+yarn install
```
Comment (simonwo) [inline ops/repo_init.sh:12]: Does this need to install yarn first?

File: .cspell/custom-dictionary.txt [hunk 5]
```diff
@@ -107,59 +107,108 @@ Acked
 acks
```
Comment (frrist) [inline .cspell/custom-dictionary.txt:107]: is this list maintained with some tooling, or manual? Which linter consumes this?

**Review by frrist — APPROVED** [review]
LGTM. But wait for someone familiar with yarn/js to give an approval

File: ops/terraform/remote_files/scripts/install-node.sh (hunk 1 above)
Comment (aronchick) [reply ops/terraform/remote_files/scripts/install-node.sh:125]: good question - yarn ships with many installs of node, but no reason not to include the install command as well.

File: ops/aws/canary/README.md (hunk 3 above)
Comment (aronchick) [reply ops/aws/canary/README.md:73]: it's a drop in replacement - it SHOULD work in most places. 

However, in this case, 'yarn' is simply a runner for the scripts in package.json - exactly the same as npm.

File: ops/repo_init.sh (hunk 4 above)
Comment (aronchick) [reply ops/repo_init.sh:12]: same as above - most modern node ships with yarn, but no reason not to do belt and suspenders!

File: .circleci/config.yml (hunk 2 above)
Comment (aronchick) [reply .circleci/config.yml:135]: just grouping more logically (imo)

File: .cspell/custom-dictionary.txt (hunk 5 above)
Comment (aronchick) [reply .cspell/custom-dictionary.txt:107]: cspell uses this - it's all manual for now. we probably should do something more formal lol.

**Pull request merged**